index.html

```diff
@@ -5,8 +5,202 @@
+    <!--font-family: 'Tapestry', cursive;-->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Tapestry&display=swap" rel="stylesheet"> 
+    <!--font-family: 'Bad Script', cursive;-->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Bad+Script&display=swap" rel="stylesheet"> 
+    <!--font-family: 'Berkshire Swash', cursive; -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Berkshire+Swash&display=swap" rel="stylesheet"> 
+    <!--font-family: 'Reenie Beanie', cursive;-->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Reenie+Beanie&display=swap" rel="stylesheet"> 
+    <!--font-family: 'Shadows Into Light Two', cursive;-->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Shadows+Into+Light+Two&display=swap" rel="stylesheet"> 
+    <!--font-family: 'Rubik Moonrocks', cursive;-->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Rubik+Moonrocks&display=swap" rel="stylesheet">
+
+    <!--font-family: 'Kdam Thmor Pro', sans-serif;-->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Kdam+Thmor+Pro&display=swap" rel="stylesheet"> 
+    
+    <!--font-family: 'Turret Road', cursive;-->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Turret+Road:wght@700&display=swap" rel="stylesheet"> 
 </head>
 <body>
+    <div class="container">
+        <header class="header">
+            <div class="weather-report">
+                <h1> 💫  Weather Report
+                <span class="city-of">
+                    for the city of
+                </span>
+                </h1>
+            </div>
+    
+            <div id="header-city">
+                ✨ Seattle ✨
+            </div>
+            <div class="emoji">
+                🌈 ☔️🌤🌦⛈❄️⛄️
+            </div>
+        </header>
+        
+        <section class="city">
+            <div >
+            <h3>City Name</h3>
```

Comment:
It looks like you skipped straight from `h1` to `h3`.  

Remember, heading tags should form a hierarchy that could be used to build a table of contents, not for styling your text.  

If you want text to be a different size just override the default `h2` style.